include/cuco/detail/static_multimap.inl

```diff
@@ -105,20 +105,17 @@ template <typename Key,
           class ProbeSequence,
           cuda::thread_scope Scope,
           typename Allocator>
-template <typename InputIt, typename Predicate>
-void static_multimap<Key, Value, ProbeSequence, Scope, Allocator>::insert_if(InputIt first,
-                                                                             InputIt last,
-                                                                             Predicate pred,
-                                                                             cudaStream_t stream)
+template <typename InputIt, typename StencilIt, typename Predicate>
+void static_multimap<Key, Value, ProbeSequence, Scope, Allocator>::insert_if_n(
```

Comment:
I didn't intend for the top level algorithm to become an `_n` algorithm. Just the kernel.